www/src/public/js/controllers/builder/main.js

```diff
@@ -232,6 +238,9 @@
             newList.baseStats.amulet = Number(each[0]);
             each.shift();
 
+            newList.baseStats.hazelnut = Number(each[0]);
```

Comment:
There are some issues with the way you're saving and loading the hazelnut stuff, we should talk

Reply:
I believe these are all resolved based on our conversations.  There is now new version checking and testing was done for both importing and exporting of v2 and v3 lists, as well as loading v2 and v3 cookies.